examples/run_var_tau.py

```diff
@@ -139,7 +136,7 @@
     fl.image(for_plot)
 var_tau_data.name(config_dict["type"] + "_" + str(config_dict["echo_counter"]))
 var_tau_data.set_prop(
-    "postproc_type", "SpinCore_var_tau_v1"
+    "postproc_type", "SpinCore_var_tau_v2"
```

Comment:
increment post processing version so that we aren't multiplying by nScans anymore